### PR TITLE
Tableau data connector

### PR DESCRIPTION
**Pull request by simonarocco**
## Briefly describe the feature or bug that this PR addresses.
- Reads from the get records endpoint as a df
- Uploads the df to google sheets (https://docs.google.com/spreadsheets/d/1yfRbMv6D8lvqGVralHHcSH5m8O4R2jywjfDnVxyotEA/edit#gid=1985491939)
- Small fix to estimate prioritization to handle multiple population groups for the primary estimate
- Changes to readme to illustrate how to set up google sheets API auth

## Describe the steps you took to test the feature/bugfix introduced by this PR.
- Ran the tableau table generator script
- Saw that the google sheets csv was successfully updated

## Does any infrastructure work need to be done before this PR can be pushed to production?
- Yes: need to authenticate with the google sheets api client (see steps in the README)
- Also need to add ANALYZE_SPREADSHEET_ID env var

### CODE REVIEW

**Review by abeljoseph — APPROVED** [review]

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 alembic==1.4.2
 aniso8601==8.0.0
 attrs==19.3.0
+boto3==1.16.63
```
Comment (atatmaja) [inline requirements.txt:4]: nit: can we remove this req

File: app/database_etl/tableau_data_connector/table_generator.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    # Clean df
+    records_df['source_id'] = records_df['source_id'].apply(lambda x: str(x))
+    records_df = records_df.fillna('')
+    records_df = records_df.replace('[', '')
+    records_df = records_df.replace(']', '')
+
+    # Upload df to google sheet
+    g_client = GoogleSheetsManager()
+    g_client.update_sheet(os.getenv('ANALYZE_SPREADSHEET_ID'), records_df)
```
Comment (atatmaja) [inline app/database_etl/tableau_data_connector/table_generator.py:31]: Don't forget to add this env var requirement to the README!

**Review by atatmaja — COMMENTED** [review]
Mostly LGTM, two tiny nits! Great work!

**Comment by atatmaja** [comment]
Also plz fix merge conflict

**Review by abeljoseph — APPROVED** [review]

**Pull request merged**